changelog/5.5.0_2022-06-20/bugfix-not-logged-out-if-oC10-backend

```diff
@@ -3,5 +3,6 @@ Bugfix: Not logged out if backend is ownCloud 10
 We've fixed an issue, where the user won't be logged out if the backend is ownCloud 10
 
 https://github.com/owncloud/web/pull/6939
+https://github.com/owncloud/web/pull/7072
```

Comment:
Not sure when this PR was last rebased but by now it needs/deserves its own changelog item :)

Reply:
Aye, cool